core/server/services/themes/middleware.js

```diff
@@ -71,15 +57,39 @@ themeMiddleware.updateTemplateData = function updateTemplateData(req, res, next)
             blog: siteData,
             site: siteData,
             labs: labsData,
-            config: themeData,
-            member: req.member
+            config: themeData
         }
     });
 
     next();
-};
+}
+
+function updateLocalTemplateData(req, res, next) {
+    // Pass 'secure' flag to the view engine
+    // so that templates can choose to render https or http 'url', see url utility
+    res.locals.secure = req.secure;
```

Comment:
Pretty sure this isn't necessarily local template data because a site should only be served on either https: OR http:

Leaving for now, it's in the localTemplateData fn because it uses data from the `req` which technically can be different.